tests/core/OperationProcessor.spec.ts

```diff
@@ -191,11 +186,10 @@ describe('OperationProcessor', async () => {
     const nextUpdateCommitmentHash = 'EiD_UnusedNextUpdateCommitmentHash_AAAAAAAAAAA';
     const updateOperationRequest = await OperationGenerator.createUpdateOperationRequest(
       didUniqueSuffix,
-      firstUpdateRevealValue,
+      signingPublicKey.jwk,
+      signingPrivateKey,
```

Comment:
I think these should be named updatePublic/PrivateKey now. But if it's not a simple rename, then forget it!

Reply:
https://github.com/decentralized-identity/sidetree/issues/730